static/assets/js/phenopolis.js

```diff
@@ -168,7 +168,7 @@ if (!PP) {
                 $('#change_password_successful').show();
                 $("#change_password_successful").text(data.success);
             },
-            error: function (xhr, msg) {
+            error: function (data, msg) {
```

Comment:
I would suggest changing this back to xhr - see the success and error params in http://api.jquery.com/jquery.ajax/

With an error, it returns a different object to that in a success block - commonly reffered to as xhr...